app/Http/Controllers/TechniqueController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Technique;
+use App\Models\Review;
+use Illuminate\View\View;
+
+class TechniqueController extends Controller
+{
+    public function index(): View
+    {
+        $viewData = [];
+        $viewData['title'] = trans('titles.home');
+        $viewData['selected'] = 'techniques';
+        $viewData['technique'] = Technique::all($columns = ['*']);
```

Comment:
Al ser un arreglo de tecnicas
```suggestion
        $viewData['techniques'] = Technique::all($columns = ['*']);
```
lo pondría así para facilitar la lectura